pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -1092,3 +1092,15 @@ def test_constructor_ambigous_dst():
     expected = ts.value
     result = Timestamp(ts).value
     assert result == expected
+
+
+def test_constructor_before_dst_switch():
```

Comment:
Can we test the example in the issue here? (i.e. that `Timestamp(Timestamp(epoch_time, tz=..))` doesn't change value)

Reply:
@mroeschke Sure!
But the value breaks only 1 nanosecond before the switch. Currently, I'm struggling to make everything else work on every azure pipeline (see comment below).

Reply:
@mroeschke 
Added to the test. Pinging you on change.